src/scripts/Update.ts

```diff
@@ -236,6 +236,18 @@ class Update implements Saveable {
             }
         }
 
+        if (this.isOlderVersion(this.saveVersion, '0.5.4')) {
+            try {
+                //Correct statistics
+                App.game.wallet.currencies[GameConstants.Currency.questPoint] += 3000;
```

Comment:
Now you're giving it to everyone instead of people who have bought it.

Reply:
This also won't add to the players Quest Points, as it's not quite initialized yet.
`App.game.update` is the first thing initialized so that all the other data can update correctly based on what we change.
You will need to modify the `saveData.wallet` object.